### DISCUSSION TITLE
I don't understand the point of Glance...

### DISCUSSION THREAD

**Discussion by tedbergstrand** [zen-browser/desktop#2464] [Q&A]
This is just a new tab that has less functionality, covers up my working tab, takes longer to appear and disappear, and is irretrievable after returning to the working tab... I don't get it. I think Glance should be thrown out in place of a hotkey to open the tab in a new split. Or Glance should be treated as a form of a split.

Temporary windows don't make sense to me. I'm perfectly capable of hitting the back/close button and Glance doesn't improve on that experience-- you still have to close the Glance window, just like you'd close a tab. The Open and Close Glance animations take considerably longer than opening/closing a new tab, and the performance in a Glance window seems to be worse than a regular tab, though this is just a feeling.

The term Glance implies speed. But this is slower than popping open a handful of new tabs in the background and hotkey closing them.

Glance just sets me up so if I stumble into something in a Glance window, I have to do extra user actions to get it to become a "permanent" tab. I think what would really let me "glance" at something is to hotkey a new split on the click that I can Ctrl+W away when I'm done with it.

I get that it's pretty, but Pretty != Zen.

Is there any use-case you guys have found where Glance is superior to a new tab/split?

**Comment by joaovl5** [comment]
just opened your issue through glance to have a quick peek at what its about. it acts as kind of a temporary view: I don't want a tab I won't be using to take more space on my sidebar. hope this helps understand the use cases.

**Reply by tedbergstrand** [reply]
Right, but you still have to close it. So it's no different than a tab. You're closing it either way. It's just slower and irretrievable. It's the same action count, but worse usability. 

What difference does it make if there's a tab open for the 30 seconds you're glancing into it? You have to close both. So whether you're closing a Glance or closing a tab, you wind up back in the same place with the same screen layout and the same number of tabs.

**Reply by joaovl5** [reply]
It's easier to close it and go back to the page, this might sound trivial but makes a ton of difference when you have to quickly peek at dozens of pages at once

**Reply by tedbergstrand** [reply]
I'm not trying to be argumentative. I'm genuinely trying to understand what you mean.

But I'm still struggling to understand the workflow difference. You can't Glance at 12 pages at once. So you open one, close it. Open the next, close it. How is that workflow different than splitting the tab, closing it, splitting the new tab, closing it... 

Isn't it the same thing? Why is it its own implementation instead of a new version of a Split that opens midscreen at 85% size and closes when it loses focus?

**Reply by CritLoren** [reply]
It doesn't change your tab view for one, so there's less risk of things messing about on your screen when you close that other tab, but the main thing I think is that it doesn't feel like you're committing to a full experience of the other page, you're just quickly checking some information with the intention of returning to the original site, this could be account settings, reading up on a specific term, or just opening the original image on discord in a pop-up without taking up an entire tab. Functionally it's the same click for click with opening a new tab, or splitting up the current screen into multiple tabs, but those change things on the screen too much for actions that are meant to be small.

**Reply by Tactile-Taco** [reply]
> Functionally it's the same click for click with opening a new tab, or splitting up the current screen into multiple tabs, but those change things on the screen too much for actions that are meant to be small.

It is not the identical click for click. Closing the glance "tab" is guaranteed to return you to the tab it was spawned from. Closing a full tab will follow its own rules for where it takes you.

**Comment by cheeeebs** [comment]
You don't have to use it. I use it because I inadvertently have a lot of tabs left opened, even if unloaded, because I use compact mode and I don't see the tab bar. It's a useful feature but it's not for everyone, but like everything in Zen Browser it's an option and isn't forced on the user. 

It's also a feature that has been requested by other users, so it's not just a few that want this and have a use for it.

**Reply by tedbergstrand** [reply]
I asked if anyone had a use-case that was superior to a tab or split and opined that it should really be a split option, instead. And that we should be able to choose which way to split it with that hotkey instead of being forced into Glance. (Glance, of course, still being an option). 

I know that people requested it. I don't want to take it away from anyone. I don't want anyone who likes it to stop using it. I just think it's poorly implemented and I don't get it in its current form.

**Comment by Tactile-Taco** [comment]
I have a use case for it which I use on another browser. When I log in to my school's portal through the school's website, the link by default opens the portal on a new tab. It is more convenient for me to have the school website pinned rather than the login portal, but working in a new tab every time I open the portal is annoying. In addition the portal has a timeout before it logs me out, so I end up needing to go back to the pinned website and login again, creating another tab... The glance feature allows me to keep the school portal contained on what is effectively the same tab as the school website. It would be more convenient for me on Zen if new tab links automatically opened in glance view, but remembering to use the hotkey is not too big of an issue.
